client/src/types.tsx

```diff
@@ -90,6 +91,14 @@ export interface IBackgrounds {
 	triangles: string;
 }
 
+export interface Ianimations {
```

Comment:
To keep with naming conventions this should be IAnimations
however there already is an IAnimation so we should change that to ITextAnimation